src/class-wc-gateway-btcpay.php

```diff
@@ -302,14 +316,6 @@ public function init_form_fields()
                     'default'     => 'no',
                     'description' => sprintf(__('Log BTCPay events, such as IPN requests, inside <code>%s</code>', 'btcpay'), wc_get_log_file_path('btcpay')),
                     'desc_tip'    => true,
-               ),
-                'notification_url' => array(
-                    'title'       => __('Notification URL', 'btcpay'),
```

Comment:
I am worried a bit about this one.

If you host btcpayserver at home, you would have a different URL here than the home URL.

The reason is that in lot's of ISP, you can't resolve a domain pointing to yourself, so you need to use internal link.
Same thing if you use Tor address as home, this change would make it impossible to work, because BTCPayServer can't send the IPN via Tor.

Reply:
I see. These situations will indeed occur. In that case it's probably better to leave the notification_url in, and make get_home_url the default.
I'll update the PR